README.md

```diff
@@ -1,92 +1,5 @@
-## :hammer: Built With
+# FCC Test Suite Template
 
-- HTML and CSS
+**Fork** this project (do not open a Pull Request) to start working on any of freeCodeCamp's responsive web design course projects. Select the project from the dropdown menu and run the tests to see which requirements your project is still missing.
 
-To get a local copy up and running follow these simple steps:
-
-1. Go to the [repository page](https://github.com/djo1975/Portfolio-setup-and-mobile-first).
-2. Press the "Code" button and copy the link.
-3. Clone it using git command `git clone <link>`.
-
-## :blue_book: Learning Objectives
-
-# Code-Reviewer
-
-Description
-In the previous lessons, you 
-got an introduction to the code review process and you 
-upgraded your HTML and CSS skills. That means that you 
-are ready to perform a code review on your code!
-
-
-IMPORTANT NOTE: Read all requirements before you start building your project.
-
-
-## 💻 Get Started
-To get a local copy up and running, follow these steps.
-
-Prerequisites
-In order to run this project you need:
-
-Project requirements
-
-
-
-Create a new empty repo.
-
-
-
-Copy your code from the project that you completed during the Admissions Trial: - If you do not have the code from 
-the Admissions Trial ask someone from your Standup Team to share their code with you. 
-In that case, remember to add an attribution note and acknowledgment in your README file.
-
-
-
-Create a new branch for your copied code.
-Open a pull request.
-
-
-Play the role of the Code Reviewer:
-Follow the list from the article "How to give a code review?".
-
-
-Check if you have used semantic HTML tags (see examples of these tags in the first bullet of the best practices document below).
-
-
-Check if you applied HTML & CSS best practices.
-Add as many comments as possible in the given time and finish your review. 
-
-Remember to add a minimum of 2 comments - you will definitely find something that could have been done better!
-
-
-Copy a link to the pull request and ask for a code review in your Student Dashboard as usual. 
-(The only difference will be that the Code Reviewer won't check your code but the review 
-you performed - isn't it a code review inception? ;))
-Work and submission mode
-You should submit this
-activity individually.
-
-
-(back to top)
-## Authors
-
-?? **Mladan**
-
-- GitHub: [@djo1975](https://github.com/djo1975)
-
-Project code author yodit93
--GitHub: https://github.com/yodit93/Judy-denyse
-
-## ?? :raised_hand: :raised_hand: Contributions
-
-Contributions, issues, and feature requests are welcome!
-
-Feel free to check the [issues page]([https://github.com/ArthurGC/portfolio-microverse/issues](https://github.com/djo1975/Portfolio-setup-and-mobile-first/issues)).
-
-## Acknowledgments
-
-- Thanks to Microverse
-- Thanks to @yodit93.
-## ?? License
-
```

Comment:
You have to implement linters.yml. to /.github)/workflows/ folder. If you need help you can fallow this link https://github.com/microverseinc/linters-config